saltstack/base/salt/unbound/init.sls

```diff
@@ -52,16 +54,41 @@ config_unbound_server:
     - source: salt://{{ slspath }}/etc/unbound/unbound.conf
     - mode: 644
 
+{% if grains['os_family'] == 'Debian' %}
+
+create_unbound_example_local_d:
+  file.directory:
+    - name: /etc/unbound/local.d/
+
+create_unbound_example_conf_d:
+  file.directory:
+    - name: /etc/unbound/conf.d/
+
+config_unbound_example_local_d:
+  file.managed:
+    - user: root
+    - group: root
+    - name: /etc/unbound/local.d/block-example.com.conf
+    - source: salt://{{ slspath }}/etc/unbound/local.d/block-example.com.conf
+    - mode: 644
+
+config_unbound_example_conf_d:
+  file.managed:
+    - user: root
+    - group: root
+    - name: /etc/unbound/conf.d/example.com.conf
+    - source: salt://{{ slspath }}/etc/unbound/conf.d/example.com.conf
```

Comment:
Example configs should not be activated by default. If you modify the file extension to anything else than `.conf` then it wouldn't be applied by unbound.

Reply:
Fixed

Reply:
Done